sky/clouds/gcp.py

```diff
@@ -29,6 +34,21 @@
 
 _IMAGE_ID_PREFIX = ('projects/deeplearning-platform-release/global/images/')
 
+_GCLOUD_INSTALLATION_LOG = '~/.sky/logs/gcloud_installation.log'
+# Need to be run with /bin/bash
+GCLOUD_INSTALLATION_COMMAND = f'pushd /tmp &>/dev/null && \
```

Comment:
Can we add a comment on why we factored this out?

Reply:
Added! Thanks!